vuln-reach/src/javascript/lang/symbol_table.rs

```diff
@@ -299,24 +308,39 @@ impl<'a> SymbolTable<'a> {
 
         let name = self.tree.repr_of(node);
 
-        // If the identifier is in a formal parameters list, the declaration
-        // scope is in the sibling "body" field block.
+        // For function parameters, the scope is the function body.
```

Comment:
I rephrased this comment to be more in line with my catch clause comment and be a little easier to understand and more concise. I do however see how it could technically be a little less generally correct.

Are there any significant places where `formal_parameters` exist outside of functions?